code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -259,6 +267,9 @@
 			user << "\blue The walls suddenly disappear."
 	return 1
 
+/datum/spellbook_entry/item/scryingorb/CanBuy(var/mob/living/carbon/human/user,var/obj/item/weapon/spellbook/book)
+	return ..() && !limit

```

Comment:
I'm not sure why this is here. The scrying orb shouldn't be touched, if you ask me.